pkg/kubelet/metrics/metrics.go

```diff
@@ -719,6 +723,24 @@ var (
 		},
 	)
 
+	MemoryManagerPinningRequestTotal = metrics.NewCounter(
```

Comment:
nit: You comment the `MemoryManagerPinningErrorsTotal` but not this one.

Reply:
Ack I'll fix it thanks

Reply:
Done!